src/Microsoft.Identity.Client/MigrationAid/MigrationAid.cs

```diff
@@ -2085,6 +2089,77 @@ public ClientCredential(string secret)
         }
 
         internal string Secret => throw MsalExceptionFactory.CreateMsalNet3BreakingChangesException();
+    }
+#endif
+
+    /// <summary>
+    /// 
+    /// </summary>
+    [Obsolete("TODO: document this. " + MsalErrorMessage.AkaMsmsalnet3BreakingChanges, true)]
```

Comment:
Proposing

In MSAL.NET 3.x, you should pass directly the Activity (on Xamarin.Android), or Window (on .NET Framework and UWP) as second parameter of <see cref="AcquireTokenInteractive"/>